pkgs/servers/gemini/stargazer/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, fetchFromSourcehut, rustPlatform, installShellFiles, scdoc, ... }:
```

```suggestion
{ lib
, fetchFromSourcehut
, rustPlatform
, installShellFiles
, scdoc
}:
```

We do not use ellipsis here.